### PR TITLE
Expose API

### PR DESCRIPTION
**Pull request by SergioRibera**
- [x] Nixification
- [x] Daily Code Challenge
- [x] Expose http API for daily challenge

### CODE REVIEW

**Review by CrawKatt — COMMENTED** [review]
se ve bien, lo apruebo. Falta la aprobación de @Phosphorus-M

**Comment by Phosphorus-M** [comment]
LGTM
boe

File: src/config/setup.rs [hunk 1]
```diff
@@ -3,122 +3,95 @@ use std::path::PathBuf;
...
-
-        let song_file_compressed = song_file.replace("mp3", "dca");
-
-        std::thread::spawn(move || {
-            let mut out_file = std::fs::File::create(&song_file_compressed).unwrap();
-            std::io::copy(&mut creator, &mut out_file).expect("Error writing out song!");
-        });
-
-        audio_map.insert(sound.into(), CachedSound::Compressed(song_src));
-}
```
Comment (Phosphorus-M) [inline src/config/setup.rs:110]: Why 🥺

Comment (SergioRibera) [reply src/config/setup.rs:110]: Too much problems 😞

**Pull request merged**